### PR TITLE
updated creating reactions documentation, added boilerplates

### PR DESCRIPTION
**Pull request by mjhea0**

### CODE REVIEW

**Comment by mjhea0** [comment]
https://assembly.com/runbook/bounties/239

**Comment by vanstee** [comment]
Lookin' good :+1:

File: docs/developers/creating_reactions.md [hunk 1]
```diff
@@ -167,12 +205,18 @@ After a Monitor process has finished performing its tasks, the Monitor will send
 
 All Reactions are modules to the actioning system, located in `actions/`. After looking up the details of a Reaction, the actioner will import the module specified by the `short-name` defined. For example, the Cloudflare Remove IP Reaction is loaded by importing `actions.cloudflare-ip-remove`. After importing the module the actioner will call the `action()` method defined in that module. The `action()` method is a keyword arguments defined method. When the actioner calls this method it specifies values for `redata`, `jdata, `rdb_server`, `r_server` and `config`.
```
Comment (dustinrc) [inline docs/developers/creating_reactions.md]: Missing backtick after jdata.

Comment (mjhea0) [inline docs/developers/creating_reactions.md]: updated! +1

**Pull request merged**